routes/ffxiv.py

```
@@ -46,3 +46,54 @@
+            "maxPurchasePrice": int(request.form.get("maxPurchasePrice")),
+            "connectedRealmIDs": {},
+        }
+
+        response = requests.post(
+            f"{api_url}/wow/shoppinglistx",
+            headers={"Accept": "application/json"},
+            json=json_data,
+        ).json()
+
+        if "data" not in response:
+            logger.error(
```

Ruff

Undefined name `logger`

(F821)